src/bin/pg_autoctl/cli_do_monitor.c

```diff
@@ -581,7 +582,8 @@ cli_do_monitor_version(int argc, char **argv)
 static void
 cli_do_monitor_parse_notification(int argc, char **argv)
```

Comment:
The comment of this function is incorrect now I think.

Reply:
Fixed.